src/common/components/header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import {Image, Flex, Button, Heading, HStack} from "@chakra-ui/react";
+
+function Header() {
+    return (
+        <HStack bg="white" h="72px" >
+            <Flex justify="space-between" w="100%" >
+                <HStack>
+                    <Image src="https://github.com/UVic-Environmental-Engineering-Club/usv-ground-station/blob/main/public/logo.png?raw=true" alignSelf="center" m="5px 5px 5px 15px" width="50px" height="50px"/>
```

Comment:
Instead of having margin 15px on the image and heading, you can move it to the parent HStack as mx (margin horizontal).
https://chakra-ui.com/docs/features/style-props